packages/react-charting/docs/TestPlans/StackedBarChart/ComponentTests.md

```diff
@@ -0,0 +1,58 @@
+|                  - onRenderCalloutPerDataPoint is not given                  |                               Should not render onRenderCalloutPerDataPoint                               |    Enzyme     |
+|               Test 3: Render calling with respective to props                |                                                                                                           |               |
+| - No prop changes: Mount stacked bar chart and then set the same props again |                               Render function should have been called twice                               |    Enzyme     |
+|   - Prop changes: Mount stacked bar chart and then set the some other prop   |                               Render function should have been called twice                               |    Enzyme     |
+|                             Test 4: Mouse events                             |                                                                                                           |               |
+|                            - Mouse over on a bar                             |                               Should render callout correctly on mouseover                                |    Enzyme     |
+|                - Mouse over on a bar with customized callout                 |                               Should render customized callout on mouseover                               |    Enzyme     |
+|                            - Mouse move on a bar                             |                               Should render callout correctly on mousemove                                |    Enzyme     |
+|        Test 5: Render empty chart aria label div when chart is empty         |                                                                                                           |               |
+|               - Stacked bar chart mounted with non-empty data                |                                  No empty chart aria label div rendered                                   |    Enzyme     |
+|                 - Stacked bar chart mounted with empty data                  |                                    Empty chart aria label div rendered                                    |    Enzyme     |
+|                     Test 6: [Sub-Component]: Stacked Bar                     |                                                                                                           |               |
+|                             - Specify bar height                             |                                Should render the bar with the given height                                |      RTL      |
+|                        - Specify bar background color                        |                           Should render the empty bars with the specified color                           |      RTL      |
```

Comment:
Why bar is empty here?

Reply:
Because if the bars are not empty (or data is not 0), the bar background color is not fixed. Example: When barBackgroundColor={DefaultPalette.red} for both the cases, 
Case1: Data > 0: 
<img width="540" alt="image" src="https://github.com/microsoft/fluentui/assets/120183316/f11175e2-1435-403b-b778-3e1c145079be">

Case 2: Data==0: 
<img width="543" alt="image" src="https://github.com/microsoft/fluentui/assets/120183316/6215afe4-c8c1-40c6-802f-1627773d56a4">